README.md

```diff
@@ -0,0 +1,22 @@
+# Gloom3D
+Gloom3D is the port of Boom3D for Haiku.
+
+![Gloom3D screenshot](gloom3d.png)
+
+
+Building
+--------
+
+Simply go into Gloom3D directory and type ``make``.
+
+
+Copyright
+------------
+
+Original 2000 version by Sasha Chukov, enhanced and updated for Haiku in 2018 by Arnav Bhatt.
+
+
+Contact
```

Comment:
Same here, you can drop the contact info.

Reply:
Done.